### ISSUE TITLE
Is this a spelling mistake?

### ISSUE THREAD

**Issue by Just-another-pleb** [Toxblh/node-red-contrib-samsung-tv-control#49]
I could be wrong, but.

I only noticed because today I was wanting this button.

`SELECT`   In the list it is spelt SEFFECT.

![Screenshot from 2020-11-27 07-04-04](https://user-images.githubusercontent.com/39888280/100388495-e3281700-307e-11eb-95c0-5c01db993126.png)

**Comment by Toxblh** [comment]
Did you check it? Because why not s-effect?

**Comment by Just-another-pleb** [comment]
Ok, fair enough.

This was seen as a problem because:

I needed to send a "confirm" (accept/enter/select) command.

On the remote control there are the 4 arrow buttons.  Up/Down/Left/Right.
In the diagonal corners there are:  WWW GUIDE RETURN and EXIT.

RETURN is confusing - not against you - to what it does.   It is maybe more a "BACK" than RETURN.  (What ever)

I needed to use the middle button at the center of this group.   It has the rectangle with the arrow in the middle.

To me that is ENTER.

I scrolled up to the `E` part and couldn't find an ENTER listed.   Hmmmm.....

I am sorry.  I have now (after typing all this) found ENTER.

Greatest apologies.
